src/__tests__/unit/bridge.handler.unit.ts

```diff
@@ -22,6 +24,15 @@ describe('Service: Bridge', () => {
     expect(lockingCap).to.be.Number();
   });
 
+  it('returns true if tx hash was processed by bridge, false if not', async () => {
```

Comment:
Please add a test using a valid btc tx for pegin sent to a previous Federation address